src/graphql/admin/schema.graphql

```diff
@@ -237,10 +257,26 @@ type BTCWallet implements Wallet {
 
 type IntraLedgerTransaction implements Transaction {
   id: ID!
+
+  """
+  From which protocol the payment has been initiated.
```

Comment:
is `protocol` really the right name?

Reply:
do you have a better name for describing onchain/lightning/intraledger?

Reply:
`bitcoin layer` perhaps?

Reply:
Hmm its probably fine - can't think of something better.